src/cross_module_fn.c

```diff
@@ -130,6 +134,16 @@ ts_dist_validate_as_data_node(PG_FUNCTION_ARGS)
 	PG_RETURN_VOID();
 }
 
+#ifndef USE_TELEMETRY
+static Datum
+error_no_telemetry(PG_FUNCTION_ARGS)
+{
+	ereport(ERROR,
+			(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+			 errmsg("this version was built without telemetry")));
```

Comment:
Why do we need to keep the cross-module function to raise an error. The point of the PR was to remove all telemetry code so why leave stubs?

Reply:
Agree. Please see [my comment](https://github.com/timescale/timescaledb/pull/3958#issuecomment-1055164641) above.